README.md

```diff
@@ -101,8 +101,11 @@ under github projects.
 
 # Definition of Ready:
 Our D.O.R. requires that the team be able to determine "what needs to be done" and the "amount of work" required to complete a given story.
-The below checklist should be complete to ensure a story is Ready.
-- [ ] The story captures and describes a reasonable unit of work
-- [ ] The amount of work to complete the story has been estimated using the approved measurement (T-shirt Size)
-- [ ] The story is in line with the Product Vision
+The below checklist should be complete to ensure a story is ready.
+- [ ] The user story has a title which the team can use to refer to the vertical slice.
+- [ ] The user story has an excellent opening sentence of the form "As a [role], I want [X] so that [Y]".
```

Comment:
I agree. I chose this wording because the assignment requirement is an "excellent" user story: 
![image](https://user-images.githubusercontent.com/9517711/86261765-d658d300-bb73-11ea-930d-8d7df8920709.png)